source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -285,6 +297,38 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
   State state_{State::Running};
   std::function<void()> yield_callback_;
   Http::AsyncClient::Request* http_request_{};
+  FireAndForgetWriter* fireAndForgetWriter_;
+};
+
+/**
+ * Implementation that takes incoming requests and implements
```

Comment:
Sorry, could you expand this a little bit? I'm a bit confused by "takes incoming requests".